tests/entrypoints/llm/test_guided_generate.py

```diff
@@ -325,3 +326,62 @@ def test_guided_json_object(llm, guided_decoding_backend: str):
             # Parse to verify it is valid JSON
             parsed_json = json.loads(generated_text)
             assert isinstance(parsed_json, dict)
+
+
+
+@pytest.mark.skip_global_cleanup
+def test_json_with_any_whitespace_disabled(llm):
+    
+    class ResponseSchema(BaseModel):
+        clarifying_question: str
+        cost_per_serving: str
+        calories: str
+        type_dish_ids: str
+        type_meal_ids: str
+        product_ids: list[str]
+        exclude_product_ids: list[str]
+        allergen_ids: list[str]
+        total_cooking_time: str
+        kitchen_ids: str
+        holiday_ids: str
+        
+    # Note: Without this setting the response will be full of `\n`
+    # for this model
+    guided_decoding_backend='xgrammar:disable-any-whitespace'
```

Comment:
It looks like the test is making an assumption about which model is being used. That could change. The test is still fine, I think. Regardless of the model, it still helps validate that '\n' never shows up, which is the red flag we care about.

```suggestion
    # Note: Without this setting, the response is sometimes full of `\n`
    # for some models. This option prevents that.
    guided_decoding_backend='xgrammar:disable-any-whitespace'
```